Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -37,9 +38,9 @@ def __line__():
     return cf.f_back.f_lineno
 
 
-# 42 - The line offset from the beggining of the file.
+# 43 - The line offset from the beggining of the file.
 _MODULES_LINE_MAPPING = {
-    'CommonServerPython': {'start': __line__() - 42, 'end': float('inf')},
+    'CommonServerPython': {'start': __line__() - 43, 'end': float('inf')},
```

Comment:
are you sure line offset should change by 1? from the diff it seems you added 4 lines:
1412,10295-10297.
What am I missing?

Reply:
The line offset is calcaulted from the beggining of the file until this line, which calls the function `__line__`.
Because I added an import which is before this line, the difference is only 1.